modules/grafana/files/dashboards/contextual_sidebar_metrics.json

```diff
@@ -0,0 +1,1137 @@
+{
+  "__inputs": [
+    {
+      "name": "DS_GRAPHITE",
+      "label": "Graphite",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "graphite",
+      "pluginName": "Graphite"
+    }
+  ],
+  "__requires": [
+    {
+      "type": "grafana",
+      "id": "grafana",
+      "name": "Grafana",
+      "version": "4.5.2"
+    },
+    {
+      "type": "panel",
+      "id": "graph",
+      "name": "Graph",
+      "version": ""
+    },
+    {
+      "type": "datasource",
+      "id": "graphite",
+      "name": "Graphite",
+      "version": "1.0.0"
+    },
+    {
+      "type": "panel",
+      "id": "singlestat",
+      "name": "Singlestat",
+      "version": ""
+    },
+    {
+      "type": "panel",
+      "id": "text",
+      "name": "Text",
+      "version": ""
+    }
+  ],
+  "annotations": {
+    "list": []
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "hideControls": false,
+  "id": null,
+  "links": [],
+  "rows": [
+    {
+      "collapse": false,
+      "height": 45,
+      "panels": [
+        {
+          "content": "These metrics display how many times a specific sidebar is displayed on content pages.\n\nThe list below is ordered in priority on which sidebar is displayed if the piece of content has multiple navigation types;\n1. Single step by step (step_by_step_nav)\n2. Mainstream browse (related_navigation)\n3. Curated related items (related_navigation)\n4. Live taxons (taxonomy_navigation)\n5. Default(related_navigation)\n ",
+          "id": 6,
+          "links": [],
+          "mode": "markdown",
+          "span": 12,
+          "title": "",
+          "type": "text"
+        }
+      ],
+      "repeat": null,
+      "repeatIteration": null,
+      "repeatRowId": null,
+      "showTitle": false,
+      "title": "Dashboard Row",
+      "titleSize": "h6"
+    },
+    {
+      "collapse": false,
+      "height": 108,
+      "panels": [
+        {
+          "cacheTimeout": null,
+          "colorBackground": false,
+          "colorValue": false,
+          "colors": [
+            "rgba(245, 54, 54, 0.9)",
+            "rgba(237, 129, 40, 0.89)",
+            "rgba(50, 172, 45, 0.97)"
+          ],
+          "datasource": "${DS_GRAPHITE}",
```

Comment:
I think you might need to replace `${DS_GRAPHITE}` with `Graphite`, or at least this is what I've done with other dashboards.

It might be worth checking that this is still necessary though, either deploy this branch of govuk-puppet to Integration, or just copy the file in to the right place, and check if works or not.

Reply:
Yep you were correct and the change has been made 👍